File: app/src/main/java/com/filemanager/fileexplorer/myfile/cleanmaster/free/Activity/customview/DefaultProgressTextAdapter.java
package com.filemanager.fileexplorer.myfile.cleanmaster.free.Activity.customview;

import com.filemanager.fileexplorer.myfile.cleanmaster.free.Activity.customview.CircularProgressIndicator;


public final class DefaultProgressTextAdapter implements CircularProgressIndicator.ProgressTextAdapter {
    @Override
    public String formatText(double d) {
        return String.valueOf((int) d);
    }
}

File: app/src/main/java/com/demo/filemanager/Activity/customview/DefaultProgressTextAdapter.kt
package com.demo.filemanager.Activity.customview

import com.demo.filemanager.Activity.customview.CircularProgressIndicator.ProgressTextAdapter

class DefaultProgressTextAdapter : ProgressTextAdapter {
    override fun formatText(d: Double): String? {
        return d.toInt().toString()
    }
}
